Content/NPCs/Moonstone/Dreambeast.cs

```diff
@@ -77,28 +78,30 @@ public override ModNPC Clone(NPC newEntity)
 			return clone;
 		}
 
-		public override void OnSpawn(IEntitySource source)
+		public override void AI()
 		{
-			for (int k = 0; k < chains.Length; k++)
+			if (!hasLoaded)
```

Comment:
put this into a helper function for readability and check against not a server: a server cannot have verlet chains since it does not have a graphics device